algorithms/ar-bsrh/javascript/binary_search.js

```diff
@@ -0,0 +1,37 @@
+function binarySearch(items, value) {
```

Comment:
Codacy found an issue: [Function 'binarySearch' has a complexity of 6. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6275956)